<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PriceList</class>
 <widget class="QWidget" name="PriceList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>465</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>372</width>
    <height>465</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>372</width>
    <height>465</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_confirm">
   <property name="geometry">
    <rect>
     <x>193</x>
     <y>435</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>281</x>
     <y>435</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>170</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>结晶</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_LYJJ">
      <property name="text">
       <string>雷云结晶：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_LYJJ">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_FHJJ">
      <property name="text">
       <string>烦惑结晶：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_FHJJ">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_TSJJ">
      <property name="text">
       <string>天寿结晶：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_TSJJ">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>125</y>
     <width>171</width>
     <height>120</height>
    </rect>
   </property>
   <property name="title">
    <string>镜</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_XYJ">
      <property name="text">
       <string>雪影镜：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_XYJ">
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_JYJ">
      <property name="text">
       <string>金牙镜：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_JYJ">
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_YHJ">
      <property name="text">
       <string>曜火镜：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_YHJ">
      <property name="maximum">
       <double>9999.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>249</y>
     <width>354</width>
     <height>178</height>
    </rect>
   </property>
   <property name="title">
    <string>其他</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="3" column="0">
     <widget class="QLabel" name="label_SGS">
      <property name="text">
       <string>神功石：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_HYG">
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_HYG">
      <property name="text">
       <string>黑月钢：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_TQLJJ">
      <property name="text">
       <string>天乾灵结晶：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_XFG">
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_XYK">
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_TZZSP">
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_XYK">
      <property name="text">
       <string>雪影矿：</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="label_TZZSP">
      <property name="text">
       <string>挑战珠碎片：</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_QYZXW">
      <property name="text">
       <string>起源信物：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_HYMNDXJ">
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_XFZTH">
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_TQLJJ">
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_XFZTH">
      <property name="text">
       <string>雪峰之铁华：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_HYMNDXJ">
      <property name="text">
       <string>黑月魔女的刑具：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_XFG">
      <property name="text">
       <string>雪峰钢：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_SGS">
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_QYZXW">
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>193</x>
     <y>125</y>
     <width>171</width>
     <height>120</height>
    </rect>
   </property>
   <property name="title">
    <string>星石</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QLabel" name="label_CHXS">
      <property name="text">
       <string>晨晖星石：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_CHXS">
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_JYXS">
      <property name="text">
       <string>金雷星石：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_JLXS">
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_YGXS">
      <property name="text">
       <string>月光星石：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_YGXS">
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>193</x>
     <y>9</y>
     <width>170</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>魂石</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="label_NTHS">
      <property name="text">
       <string>南天魂石：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_NTHS">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_MHHS">
      <property name="text">
       <string>冥幻魂石：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_MHHS">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>17</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_clear">
   <property name="geometry">
    <rect>
     <x>107</x>
     <y>435</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>清除</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_close</sender>
   <signal>clicked()</signal>
   <receiver>PriceList</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
